thold_functions.php

```diff
@@ -3596,7 +3634,7 @@ function get_email_subject($phase, $trigger, $lastread, $ra, $breach_up, &$thold
 					' changed to Warning Threshold with value ' .
 					thold_format_number($thold_data['lastread'], $decimals, $baseu, $suffix, $show_units, $units_suffix);
 			} else {
-				return $phase . ':' . thold_get_cached_name($thold_data) .
```

Comment:
Why adding the space?

Reply:
The space was added to the alert to match the other notifications, normal and warning.

Reply:
Okay, this is resolved upstream.